style.css

```diff
@@ -56,13 +58,22 @@ p.button-text {
     margin-right: auto;
 }
 
-h3 {
+h3.black {
     color: #000;
     font-size: 2.5rem;
     font-family: 'Cormorant Upright','Georgia',serif;
```

Comment:
Since this font-family set is used a few times, you could consider using CSS variables. :) I can see gold-yellow already declared and working, nice job! 🏅